### PR TITLE
Implement Paste without formatting/indentation

### PR DESCRIPTION
**Pull request by trestoa**
### Description of the Change

This PR implements Paste without formatting/indentation as described in https://github.com/atom/atom/issues/5046. Currently, the `autoIndent` property of the `options` parameter is always overwritten by the `autoIndentOnPaste` config value. With this PR, the property is only assigned if it not present.

Also, it adds an menu entry for Paste without formatting.

### Benefits

As discussed in https://github.com/atom/atom/issues/5046.

### Applicable Issues

Implements https://github.com/atom/atom/issues/5046.

### CODE REVIEW

File: src/register-default-commands.coffee [hunk 1]
```diff
@@ -174,6 +174,7 @@ module.exports = ({commandRegistry, commandInstaller, config, notificationManage
         'core:cut': -> @cutSelectedText()
         'core:copy': -> @copySelectedText()
         'core:paste': -> @pasteText()
+        'core:paste-verbatim': -> @pasteText autoIndent: false
```
Comment (50Wliu) [inline src/register-default-commands.coffee]: Can this call be wrapped in parentheses?

Comment (kleiinnn) [inline src/register-default-commands.coffee]: Yes, of course. Will do that.

**Comment by msurguy** [comment]
Really need this merged :)

**Comment by HaykoKoryun** [comment]
@msurguy whenever I need to paste verbatim I have to switch to Notepad++, paste the text in and remember not to touch the file again just in case.

**Comment by sadick254** [comment]
Hey @trestoa. Apologies for not looking into this on time. I know this PR has been here for a very long time, during that time Atom has seen a number of changes. Due to the changes, this PR has now become outdated. Is it possible for you to resolve the conflicts?

**Comment by sadick254** [comment]
Thank you for your PR!

We haven't gotten a response to our conflict resolution request. I'm going to close this but don't hesitate to reach out if you have or find time to resolve the conflicts, we'll be happy to reopen the PR.

**Pull request closed without merging**